data_model/src/events/data/events.rs

```diff
@@ -306,6 +297,42 @@ mod config {
     }
 }
 
+mod validator {
+    use super::*;
+
+    model! {
```

Comment:
why is it not a `data_event!` with `HasOrigin`?

Reply:
Because it doesn't have and origin. `Validator` has no id. But our `data_event!` derives `HasOrigin`. So I just implemeted this by hand as an exception